spec/models/ticity_search_spec.rb

```diff
@@ -1,5 +1,84 @@
 require 'rails_helper'
 
 RSpec.describe TicitySearch, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it "should validate city1" do
+    search = TicitySearch.create
```

Comment:
Pull this into the subject for your test: https://relishapp.com/rspec/rspec-core/v/2-0/docs/subject/explicit-subject.

Then you can just say `expect(subject.errors[:key]).to_not be_empty`.

Also, turn the list of attributes into a constant, then you can do something like:

```
TicitySearch::SEARCH_ATTRIBUTES.map do |attr|
      expect(subject.errors[attr]).to_not be_empty
end
```
You can also pass that constant to `validates` so you don't have to type it twice